mlflow/ml-package-versions.yml

```diff
@@ -419,3 +419,35 @@ diviner:
       "<= 0.1.0": ["pmdarima<2.0.0"]
     run: |
       pytest tests/diviner/test_diviner_model_export.py
+
+h2o:
+  package_info:
+    pip_release: "h2o"
+  models:
+    minimum: "3.40.0.1"
+    maximum: "3.40.0.1"
```

Comment:
Chose the current latest version.

Reply:
With how frequently they release, we might want to think about keeping a narrow band between min and max so that we're not running too many nightly cross versions tests :)

Reply:
Agreed, I think we should also consider setting a lower limit (e.g. version released within two years).